Library/Formula/gnu-barcode.rb

```diff
@@ -1,14 +1,21 @@
 class GnuBarcode < Formula
   desc "Convert text strings to printed bars"
   homepage "https://www.gnu.org/software/barcode/"
-  url "http://ftpmirror.gnu.org/barcode/barcode-0.98.tar.gz"
-  mirror "https://ftp.gnu.org/gnu/barcode/barcode-0.98.tar.gz"
-  sha256 "4229e19279b9787ac7e98852fa0bfd93986dce93b9cb07d93a017d68d409b635"
+  url "http://ftpmirror.gnu.org/barcode/barcode-0.99.tar.gz"
+  mirror "https://ftp.gnu.org/gnu/barcode/barcode-0.99.tar.gz"
+  sha256 "7c031cf3eb811242f53664379aebbdd9fae0b7b26b5e5d584c31a9f338154b64"
+
+  # Patch and ac_cv_func_calloc_0_nonnull config addresses the following issue:
+  # https://github.com/Homebrew/homebrew/issues/41115
```

Comment:
Sorry, I meant could you link to the [upstream submission](https://lists.gnu.org/archive/html/bug-barcode/2015-06/msg00000.html), rather than our bug report. Has upstream got back to you on moving forwards with it yet, out of curiosity?